src/base-command.ts

```diff
@@ -29,6 +43,119 @@ export default abstract class BaseCommand extends Command {
     });
   }
 
+  async _filterNonSensitiveSentryMetadata(non_sensitive: Set<string>, metadata: any): Promise<any> {
```

Comment:
Can we move all of this sentry logic into its own class that gets instantiated in the BaseCommand constructor kind of like the app/api?

Reply:
Moved the sentry logic out of BaseCommand and into a new class, SentryService.